src/period.c

```diff
@@ -19,9 +19,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-#ifndef _GNU_SOURCE
-#define _GNU_SOURCE
```

Comment:
This was added by commit 50d111a859e0e995bcbab57cb4fcfee3ab5ebf43 because round() requires _GNU_SOURCE on some old systems.

Reply:
OK, fixed.